apps/meteor/client/views/marketplace/AppDetailsPage/AppDetailsPageTabs.tsx

```diff
@@ -51,7 +51,7 @@ const AppDetailsPageTabs = ({
 					{t('Security')}
 				</Tabs.Item>
 			)}
-			{marketplace !== false && (
+			{marketplace !== false && context !== 'private' && (
```

Comment:
I think this marketplace test is not necessary, this prop always came as undefined... 🤔 

I suggest to check the necessity and try to find what is happening. 

If the test is from a boolean, why the type is unknown?